modules/nodes-nodes-resources-configuring-about.adoc

```diff
@@ -29,65 +28,35 @@ introduction of allocatable resources.
-can lead to critical system services being CPU starved or OOM killed on the node. The
-recommendation is to enforce system-reserved only if operators have profiled their nodes
-exhaustively to determine precise estimates and are confident in their ability to
-recover if any process in that group is OOM killed.
-
-As a result, we strongly recommended that users only enforce node allocatable for
-`pods` by default, and set aside appropriate reservations for system daemons to maintain
-overall node reliability.
+The node is able to limit the total amount of resources that pods can consume based on the configured allocatable value. This feature significantly improves the reliability of the node by preventing pods from using CPU and memory resources that are needed by system services such as the container runtime and node agent. To improve node reliability, administrators should reserve resources based on a target for resource use.
+
+The node enforces resource constraints using a new cgroup hierarchy that enforces quality of service. All pods are launched in a dedicated cgroup hierarchy that is separate from system daemons.
+
+Administrators should treat system daemons similar to pods that have a guaranteed quality of service. System daemons can burst within their bounding control groups and this behavior must be managed as part of cluster deployments. Reserve CPU and memory resources for system daemons by specifying the amount of CPU and memory resources to reserve in `system-reserved`.
+
+Enforcing `system-reserved` limits can prevent critical system services from receiving CPU and memory resources. As a result, a critical system service can be ended by the out-of-memory killer. The recommendation is to enforce `system-reserved` only if you have profiled the nodes exhaustively to determine precise estimates and you are confident that critical system services can recover if any process in that group is ended by the out-of-memory killer.
+
+As a result, we strongly recommended that you only enforce node allocatable for `pods` by default, and set aside appropriate reservations for system daemons to maintain node reliability.
 
 [id="allocate-eviction-thresholds_{context}"]
 == Understanding Eviction Thresholds
 
-If a node is under memory pressure, it can impact the entire node and all pods running on
-it.  If a system daemon is using more than its reserved amount of memory, an OOM
-event may occur that can impact the entire node and all pods running on it.  To avoid
-(or reduce the probability of) system OOMs the node provides out-of-resource handling.
+If a node is under memory pressure, it can impact the entire node and all pods running on the node. If a system daemon uses more than its reserved amount of memory, an out-of-memory event can occur that impacts the entire node and all pods running on the node. To avoid or reduce the probability of system out-of-memory events, the node provides out-of-resource handling.
```

Comment:
I wonder if we can tighten these two sentences to remove the redundancy (impacts the entire node and all pods running on the node) ? Something like: 
_If a node is under memory pressure, it can impact the entire node and all pods running on the node, for example an out-of-memory event due to a system daemon using more than its reserved amount of memory._

Reply:
That is awkward.  Thanks for pointing it out.  How's this?

> If a node is under memory pressure, it can impact the entire node and all pods running on the node. For example, a system daemon that uses more than its reserved amount of memory can trigger an out-of-memory event.